src/test/java/org/gitlab4j/api/TestUserApi.java

```diff
@@ -214,13 +214,13 @@ public void testSudoAsUser() throws GitLabApiException {
     public void testCreateImpersonationToken() throws GitLabApiException, ParseException {
 
         User user = gitLabApi.getUserApi().getCurrentUser();
-        Scope[] scopes = {Scope.API, Scope.READ_USER};
+        Scope[] scopes = {Scope.API, Scope.READ_USER, Scope.READ_REGISTRY, Scope.WRITE_REPOSITORY, Scope.SUDO};
         Date expiresAt = ISO8601.toDate("2018-01-01T00:00:00Z");
         ImpersonationToken token = gitLabApi.getUserApi().createImpersonationToken(user.getId(), TEST_IMPERSONATION_TOKEN_NAME, expiresAt, scopes);
         assertNotNull(token);
         assertNotNull(token.getId());
         assertEquals(TEST_IMPERSONATION_TOKEN_NAME, token.getName());
-        assertEquals(2, token.getScopes().size());
+        assertEquals(5, token.getScopes().size());
 
```

Comment:
@PhilippeVienne 
As tests are modified and added I have been trying to make them a bit more robust, to that end could you change this test to the following?
```
    @Test
    public void testCreateImpersonationToken() throws GitLabApiException, ParseException {

        // Arrange
        User user = gitLabApi.getUserApi().getCurrentUser();
        Scope[] scopes = {Scope.API, Scope.READ_USER, Scope.READ_REGISTRY, Scope.WRITE_REPOSITORY, Scope.SUDO};
        Date expiresAt = ISO8601.toDate("2019-01-01T00:00:00Z");

        ImpersonationToken token = null;
        try {
            
            // Act
            token = gitLabApi.getUserApi().createImpersonationToken(user, TEST_IMPERSONATION_TOKEN_NAME, expiresAt, scopes);
 
            // Assert
            assertNotNull(token);
            assertNotNull(token.getId());
            assertEquals(TEST_IMPERSONATION_TOKEN_NAME, token.getName());
            assertEquals(scopes.length, token.getScopes().size());
            assertThat(token.getScopes(), containsInAnyOrder(scopes));
            assertEquals(expiresAt.getTime(), token.getExpiresAt().getTime());

        } finally {
            if (user != null && token != null) {
                gitLabApi.getUserApi().revokeImpersonationToken(user, token.getId());
            }
        }
```

You'll also need to add the following imports:
```
import static org.hamcrest.Matchers.containsInAnyOrder;
import static org.junit.Assert.assertThat;
```

Reply:
@PhilippeVienne 
I'm working on getting a release done, so I'm going to go ahead and merge this, I will update the test per my request after merging.

Thanks again for your contribution.